cmd/helmify/flags.go

```diff
@@ -68,7 +68,7 @@ func ReadFlags() config.Config {
 	flag.BoolVar(&result.FilesRecursively, "r", false, "Scan dirs from -f option recursively")
 	flag.BoolVar(&result.OriginalName, "original-name", false, "Use the object's original name instead of adding the chart's release name as the common prefix.")
 	flag.Var(&files, "f", "File or directory containing k8s manifests")
-
+	flag.BoolVar(&preservens, "preserve-ns", false, "Use the object's original namespace instead of adding all the resources to a common namespace")
```

Comment:
overall looks good. can you please add flag to a table in [README.md](https://github.com/arttor/helmify/blob/main/README.md?plain=1#L112)?

Reply:
Added. Thanks for the feedback.